myst_nbs/diagnostics_and_criticism/sampler-stats.myst.md

```diff
@@ -57,7 +54,7 @@ with model:
     idata = pm.sample(2000, tune=1000, init=None, step=step, chains=4)
 ```
 
-- `Note`: NUTS provides the following statistics( these are internal statistics that the sampler uses, you don't need to do anything with them when using PyMC3, to learn more about them, [check this page](https://docs.pymc.io/api/inference.html#module-pymc3.step_methods.hmc.nuts).
+- `Note`: NUTS provides the following statistics (these are internal statistics that the sampler uses, you don't need to do anything with them when using PyMC, to learn more about them, [`pymc.NUTS`](https://www.pymc.io/projects/docs/en/latest/api/generated/pymc.NUTS.html#pymc.NUTS).
```

Comment:
```suggestion
- `Note`: NUTS provides the following statistics (these are internal statistics that the sampler uses, you don't need to do anything with them when using PyMC, to learn more about them, {class}`pymc.NUTS`
```

Note: should probably do the change locally in the notebook, not accept the suggestion here.